site/layouts/partials/head.html

```diff
@@ -0,0 +1,22 @@
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+  {{ partial "title.html" . }}
+  {{ $stylesheet := .Site.Data.webpack.main }}
+  <link href="{{ partial "webpack_url.html" $stylesheet.css }}" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500&display=swap" rel="stylesheet">
+  <style>
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url({{ partial "webpack_url.html" "/material-design-icons/MaterialIcons-Regular.eot" }}); /* For IE6-8 */
+  src: local('Material Icons'),
+    local('MaterialIcons-Regular'),
+    url({{ partial "webpack_url.html" "/material-design-icons/MaterialIcons-Regular.woff2" }}) format('woff2'),
+    url({{ partial "webpack_url.html" "/material-design-icons/MaterialIcons-Regular.woff" }}) format('woff'),
+    url({{ partial "webpack_url.html" "/material-design-icons/MaterialIcons-Regular.ttf" }}) format('truetype');
+}
```

Comment:
it looks like this is adding 4 new requests to each page load for these resources, do you think it's possible we could bundle these into the webpack style build instead to get all the styling in one request?

Reply:
I'm not sure, maybe. I'll investigate

Reply:
As far as I can tell this isn't adding 4 new requests because the `format` part informs the browser what's available and it picks the one which is applicable (usually woff2 for modern browsers or woff for IE). The `local` part also means it uses whatever was previously cached, so I think keeping it separate from the other parts of the bundle would be best for bandwidth.

Reply:
hmm I saw 16 requests for course home pages vs 12 on master in the chrome inspector - possibly there's a browser configuration thing going on there though? :woman_shrugging: